middleware.ml

```diff
@@ -147,3 +173,41 @@ let is_user_admin_middleware api_meth now users handler reqd =
             "You don't have the necessary permissions to access this service."
           `Unauthorized user 401 api_meth reqd ()
   | Error (`Msg msg) -> redirect_to_login ~msg reqd ()
+
+let csrf_match ~input_csrf ~check_csrf =
+  String.equal (Utils.Json.clean_string input_csrf) check_csrf
```

Comment:
Why is "clean_string" applied here? I still don't quite understand the purpose of `Utils.Json.clean_string`.

Reply:
For some reason when I get a specific json field it is quoted and String.equal would give a failed match

Reply:
Hmm, ok. Maybe we should open an issue to re-evaluate the `clean_string` and String.trim usages?